.github/workflows/validate.yml

```diff
@@ -2,32 +2,42 @@ name: validate-ci
 on:
   pull_request:
     types: [labeled, unlabeled, opened, reopened, edited, ready_for_review, synchronize]
-    branches-ignore:
-      - master
-      - '!*'
   push:
     branches-ignore:
       - master
       - '*'
```

Comment:
dont push on master?

Reply:
On master push we only run publish.yml workflow. Here i am making sure validate.yml does not run when a push is being made on master.